src/System.Devices.Gpio/Windows10Driver.cs

```diff
@@ -0,0 +1,343 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Threading;
+
+using GpioPinValueChangedHandler = global::Windows.Foundation.TypedEventHandler<global::Windows.Devices.Gpio.GpioPin, global::Windows.Devices.Gpio.GpioPinValueChangedEventArgs>;
+using WinGpio = global::Windows.Devices.Gpio;
+
+namespace System.Devices.Gpio
+{
+    public class Windows10Driver : GpioDriver
```

Comment:
Why is it public?

Reply:
Since the Windows10Driver can be created from a different *Windows*.Devices.GpioController, the class needs to have a constructor that specifies _which_ controller to adapt to the .NET Core API. Windows supports multiple GPIO devices.